Microsoft.Toolkit.Uwp.UI.Controls/TextToolbar/TextToolbar.Properties.cs

```diff
@@ -133,6 +133,11 @@ public TextToolbarStrings Labels
         /// </summary>
         public VirtualKey LastKeyPress { get; private set; }
 
+        /// <summary>
+        /// Enables use of URI Checker for Link Creator.
+        /// </summary>
+        public bool UseURIChecker { get; set; } = true;
```

Comment:
Any reason why this is not a Dependency Property? I would also keep the default false, to make sure the behavior is the same for those updating from previous version.

Reply:
Does it need to be? What benefit would a DP have here? It won't have any designer modifications.

Reply:
Scratch that, googled it: https://stackoverflow.com/questions/1723756/why-dependency-properties
I'll fix it.